docs/mapping/freiburg.md

```diff
@@ -0,0 +1,74 @@
+
+## ParkingSites
+
+A `ParkingSites` provides realtime data for a `ParkingSite`.
+
+| Field                      | Type                     | Cardinality | Mapping                         | Comment                                                             |
+|----------------------------|--------------------------|-------------|---------------------------------|---------------------------------------------------------------------|
+| obs_parkid                 | integer                  | 1           | uid                             |                                                                     |
+| obs_max                    | integer                  | ?           | realtime_capacity/capacity      |                                                                     |
+| obs_free                   | integer                  | ?           | realtime_free_capacity          |                                                                     |
+| obs_ts                     | datetime                 | 1           | realtime_data_updated_at        |                                                                     |
+| obs_state                  | integer                  | ?           | #RealtimeOpeningStatus          |                                                                     |
+| public_url                 | string                   | ?           | public_url                      |                                                                     |
+
+
+## ParkingSites for P+R Static
+
+Attributes which are set statically:
+* `has_realtime_data` is always set to `False`
+
+A `ParkingSites` provides static data for a Park and Ride `ParkingSite`.
+
+| Field                      | Type                     | Cardinality | Mapping                         | Comment                                                             |
+|----------------------------|--------------------------|-------------|---------------------------------|---------------------------------------------------------------------|
+| ogc_fid                    | integer                  | 1           | uid                             |                                                                     |
+| kapazitaet                 | integer                  | 1           | capacity                        |                                                                     |
+| name                       | string                   | 1           | name                            |                                                                     |
+| nummer                     | string                   | ?           | name                            |                                                                     |
+| kategorie                  | string                   | 1           | #ParkingSiteType                |                                                                     |
```

Comment:
The Type is a enumeration `ParkingSiteType` which should be linked. The mapping is to `type`.